packages/aws-cdk-lib/aws-ecs-patterns/lib/base/application-multiple-target-groups-service-base.ts

```diff
@@ -267,6 +277,26 @@ export interface ApplicationTargetProps {
    * @default No path condition
    */
   readonly pathPattern?: string;
+
+  /**
+   * The amount of time for Elastic Load Balancing to wait before deregistering a target.
+   *
+   * The range is 0-3600 seconds.
+   *
+   * @default 300
+   */
+  readonly deregistrationDelay?: Duration;
+  /**
+   * The stickiness cookie expiration period.
+   *
+   * Setting this value enables load balancer stickiness.
+   *
+   * After this period, the cookie is considered stale. The minimum value is
+   * 1 second and the maximum value is 7 days (604800 seconds).
+   *
+   * @default Duration.days(1)
```

Comment:
This should actually be `Stickiness is disabled`. See https://github.com/aws/aws-cdk/pull/29726